forte/data/extractor/seqtagging_extractor.py

```diff
@@ -0,0 +1,203 @@
+# Copyright 2020 The Forte Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This file implements BioSeqTaggingExtractor, which is used to extract feature
+from the tagging label.
+"""
+import logging
+from typing import List, Dict, Union
+from ft.onto.base_ontology import Annotation, EntityMention
+from forte.common.configuration import Config
+from forte.data.data_pack import DataPack
+from forte.data.converter.feature import Feature
+from forte.data.extractor.utils import bio_tagging
+from forte.data.extractor.base_extractor import BaseExtractor
+
+logger = logging.getLogger(__name__)
+
+__all__ = [
+    "BioSeqTaggingExtractor"
+]
+
+
+class BioSeqTaggingExtractor(BaseExtractor):
+    r"""BioSeqTaggingExtractor will the feature by performing BIO encoding
+    for the attribute of entry, aligining to the based_on entry.
+
+    Args:
+        config: An instance of `Dict` or
+            :class:`forte.common.configuration.Config`
```

Comment:
See `default_configs` for the expected keys.

Reply:
Got it.